crypto/bio/bf_lbuf.c

```diff
@@ -53,6 +55,7 @@ typedef struct bio_linebuffer_ctx_struct {
     char *obuf;                 /* the output char array */
     int obuf_size;              /* how big is the output buffer */
     int obuf_len;               /* how many bytes are in it */
+    int prefix_len;             /* how many bytes the prefix occupy */
```

Comment:
size_t?

Reply:
I was afraid of mixing types, but now that I look again, I should really be able to change all three to size_t without worry.  Does that seem right to you?

Reply:
Yes, size_t for all three seems correct to me.